### ISSUE TITLE
Consider refactoring browser-specific capabilities on creating a new session

### ISSUE THREAD

**Issue by fluffysquirrels** [fluffysquirrels/webdriver_client_rust#16]
As originally discussed on PR #9.

@equalsraf [wrote](https://github.com/fluffysquirrels/webdriver_client_rust/commit/66414dd9df45c971a00fa0f37f5d83822127682d#commitcomment-27204477):

> Initially I meant to set the option in the chrome driver. My question here is what to do when connecting to without the driver type. For example www can attach to a running http server without the spawning the driver.

**Comment by fluffysquirrels** [comment]
Good point. My use case for attaching was to re-use an existing session, which of course already had capabilities specified when it was created.

I'm not 100% happy with leaving browser-specific initialisation in, but it shouldn't hurt. I'm probably 95% happy with it :) . Maybe I'll add some comments to each driver and to each of the Chrome hacks in the shared code.

**Comment by equalsraf** [comment]
Well this particular option is pretty tricky because it ensures chromedriver is compatible with the w3c spec. And I dont think there is any command line option to enable this globally in chrome driver.

**Comment by fluffysquirrels** [comment]
Yup, let's leave it in for now.
